pages/store/[nftClassId]/index.vue

```diff
@@ -1216,6 +1216,7 @@ async function handlePurchaseButtonClick() {
         utmSource: 'upsell_plus',
         utmCampaign: `upsell_plus_${nftClassId.value}`,
         utmMedium: 'product_page',
+        from: getRouteQuery('from') as string | undefined,
```

Comment:
do you mean this? as `getRouteQuery()` always returns a string
```suggestion
        from: getRouteQuery('from') || undefined,
```